### PR TITLE
Elaborated text on end-to-end security

### PR DESCRIPTION
**Pull request by OliverPfaff**
Elaborated text on end-to-end security: 1 additional section plus 1 clarification for the existing section

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Mar 9, 2020, 7:58 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2Fw3c%2Fwot-security%2Febfcb5768c29f9813f2c0e5fa71316f5b3466aee%2Findex.html%3FisPreview%3Dtrue)

```
Navigation timeout of 30000 ms exceeded
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/wot-security%23164.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2371,6 +2371,24 @@ <h2>Secure Practices for designing levels below the WoT Runtime</h2>
...
+		    perception of WoT security is to support users of WoT security in implementing 
+		    security architectures according their perception of the ends (which can vary 
+		    across users, deployments etc.)
+		    For instance: MACSec provides E2E security between endpoints in one (local) 
+		    network, IPSec provides E2E security between endpoints in an IP-based internetwork, 
+		    (D)TLS provides E2E security between client and server processes, CMS as well as 
+		    XML/JSON/CBOR security can provide E2E security on level of application data
+            </p>
+	    <p>
+		    The following text addresses E2E security in the sense of protecting application
```
Comment (mmccool) [inline index.html]: I would change "text" to "technologies", in that these are examples of how to support certain forms of protection for certain definitions of "ends".  Maybe some of the above (MACSec, etc.) can be moved to a common "examples" sub-section (not a section, but perhaps a bulleted list?).  Each example should define what the "ends" are and what is being protected.

Comment (mmccool) [reply index.html]: In short, we need to integrate the existing text into the discussion, not end with "and here's some other stuff"

File: index.html [hunk 2]
```diff
@@ -2382,8 +2400,7 @@ <h2>Secure Practices for end-to-end security</h2>
 	      <a href="https://datatracker.ietf.org/doc/rfc7515/">RFC 7515 (IETF JSON Web Signature)</a>
 	      provides guidelines for computation
               and validation of digital signatures or MACs using JSON-based data structures. 
-            </p>
-            <p>
+
```
Comment (mmccool) [inline index.html]: probably should structure all these examples into a bulleted list rather than merging them into one big and hard to read paragraph.

**Comment by mmccool** [comment]
Decided in March 23 Security TF to hold off on merge while @OliverPfaff makes some editorial changes suggested by the review.

**Comment by OliverPfaff** [comment]
Could not find an obvious way to modify the to-be-merged artifact. Unless this was overseen, I suggest:
- To integrate by accepting the PR based on the present to-be-merged artifact
- Trigger it's refinement by a subsequent issue/(new) PR

**Pull request merged**